src/lib/helpers/propTypes.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * until https://github.com/facebook/prop-types/issues/263 fixed
```

Comment:
Can you also add from where you took it?

Reply:
👍